lnrpc/lightning.swagger.json

```
@@ -57,6 +57,15 @@
             }
           }
         },
+        "parameters": [
+          {
+            "name": "account",
+            "description": "The wallet account the balance is shown for.\nIf this is not specified, the balance of the default account is shown.",
```

```suggestion
            "description": "The wallet account the balance is shown for.\nIf this is not specified, the balance of the \"default\" account is shown.",
```

since the default account is actually named "default" instead of "" under the hood, it may be helpful to provide that hint here.